packages/kbn-ftr-common-functional-ui-services/services/remote/__tests__/remote_default_window_size.js

```diff
@@ -9,9 +9,12 @@
 import { fork } from 'child_process';
 
 import expect from '@kbn/expect';
+import { REPO_ROOT } from '@kbn/repo-info';
 
-const FTR_SCRIPT = require.resolve('../../../../../scripts/functional_test_runner');
-const CONFIG_PATH = require.resolve('./fixtures/several_nested_window_size_changes/config.js');
+const FTR_SCRIPT = require.resolve(`${REPO_ROOT}/scripts/functional_test_runner`);
+const CONFIG_PATH = require.resolve(
+  `${REPO_ROOT}/packages/kbn-ftr-common-functional-ui-services/services/remote/__tests__/fixtures/several_nested_window_size_changes/config.js`
```

Comment:
What's the reason to move this from a relative path to an absolute path? Asking because `./fixtures/several_nested_window_size_changes/config.js` looks much simpler then `${REPO_ROOT}/packages/kbn-ftr-common-functional-ui-services/services/remote/__tests__/fixtures/several_nested_window_size_changes/config.js`.

Reply:
There was an error.  Let me revert and see if an error comes up again.  I cant recall what it is at the moment.

Reply:
@pheyos here's an example of what I'm suggesting:
https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/functional/services/deployment_agnostic_services.ts#L9

Reply:
The no boundaries crossing lint error is not quite correct there as no actual import is happening and it's all happening within the same test directory. However, we decided to stay with the current `REPO_ROOT` based version.